client/state/purchases/selectors.js

```diff
@@ -59,23 +62,31 @@ export const getByPurchaseId = ( state, purchaseId ) =>
 
 /**
  * Returns a list of Purchases associated with a Site from the state using its siteId
- * @param  {Object} state       global state
- * @param  {Number} siteId      the site id
- * @return {Object} the matching purchases if there are some
+ *
+ * @param   {object} state       global state
+ * @param   {number} siteId      the site id
+ * @returns {object} the matching purchases if there are some
  */
 export const getSitePurchases = ( state, siteId ) =>
 	getPurchases( state ).filter( purchase => purchase.siteId === siteId );
 
-/***
+export const siteHasBackupProductPurchase = ( state, siteId ) => {
```

Comment:
How about adding a doc block here, similarly to other selectors here?

Reply:
Good call, added in https://github.com/Automattic/wp-calypso/pull/38316/commits/c13c1a7817e7d31f43b6cdcb7cd1512d7a759597